### PR TITLE
feat: add option to override the auth well known suffix

### PR DESCRIPTION
**Pull request by eneajaho**
This PR adds the possibility to override the default well know suffix.

### CODE REVIEW

**Comment by FabianGosebrink** [comment]
Hey Enea :) Nice to see a PR from you!

**Comment by eneajaho** [comment]
> Hey Enea :) Nice to see a PR from you!

Saw that I couldn't change it for my usecase, and needed to send this one PR 😄

**Comment by FabianGosebrink** [comment]
That is awesome, thank you!

File: projects/angular-auth-oidc-client/src/lib/config/openid-configuration.ts [hunk 1]
```diff
@@ -18,6 +18,13 @@ export interface OpenIdConfiguration {
   authWellknownEndpointUrl?: string;
   authWellknownEndpoints?: AuthWellKnownEndpoints;
 
+    /** 
+     * Override the default Security Token Service wellknown endpoint postfix. 
+     * 
+     * @default /.well-known/openid-configuration
+     */
+  authWellknownUrlSuffix?: string;
```
Comment (timdeschryver) [inline projects/angular-auth-oidc-client/src/lib/config/openid-configuration.ts:26]: Can you also include this in the docs please? https://github.com/damienbod/angular-auth-oidc-client/blob/main/docs/site/angular-auth-oidc-client/docs/documentation/configuration.md

Comment (eneajaho) [reply projects/angular-auth-oidc-client/src/lib/config/openid-configuration.ts:26]: Done.

**Pull request merged**